.editorconfig

```diff
@@ -5,260 +5,30 @@ charset = utf-8
-# ij_java_blank_lines_after_imports = 1
-# ij_java_blank_lines_after_package = 1
-# ij_java_blank_lines_around_class = 1
-# ij_java_blank_lines_around_field = 0
-# ij_java_blank_lines_around_field_in_interface = 0
-# ij_java_blank_lines_around_initializer = 1
-# ij_java_blank_lines_around_method = 1
-# ij_java_blank_lines_around_method_in_interface = 1
-# ij_java_blank_lines_before_class_end = 0
-# ij_java_blank_lines_before_imports = 1
-# ij_java_blank_lines_before_method_body = 0
-# ij_java_blank_lines_before_package = 0
-# ij_java_block_brace_style = end_of_line
-# ij_java_block_comment_at_first_column = true
 ij_java_call_parameters_new_line_after_left_paren = true
-# ij_java_call_parameters_right_paren_on_new_line = false
 ij_java_call_parameters_wrap = on_every_item
-# ij_java_case_statement_on_separate_line = true
-# ij_java_catch_on_new_line = false
-# ij_java_class_annotation_wrap = split_into_lines
-# ij_java_class_brace_style = end_of_line
 ij_java_class_count_to_use_import_on_demand = 9999
-# ij_java_class_names_in_javadoc = 1
-# ij_java_do_not_indent_top_level_class_members = false
-# ij_java_do_not_wrap_after_single_annotation = false
-# ij_java_do_while_brace_force = never
-# ij_java_doc_add_blank_line_after_description = true
 ij_java_doc_add_blank_line_after_param_comments = true
 ij_java_doc_add_blank_line_after_return = true
-# ij_java_doc_add_p_tag_on_empty_lines = true
 ij_java_doc_align_exception_comments = false
 ij_java_doc_align_param_comments = false
 ij_java_doc_do_not_wrap_if_one_line = true
 ij_java_doc_enable_formatting = true
-# ij_java_doc_enable_leading_asterisks = true
 ij_java_doc_indent_on_continuation = true
 ij_java_doc_keep_empty_lines = true
-# ij_java_doc_keep_empty_parameter_tag = true
-# ij_java_doc_keep_empty_return_tag = true
-# ij_java_doc_keep_empty_throws_tag = true
-# ij_java_doc_keep_invalid_tags = true
-# ij_java_doc_param_description_on_new_line = false
 ij_java_doc_preserve_line_breaks = false
-# ij_java_doc_use_throws_not_exception_tag = true
-# ij_java_else_on_new_line = false
-# ij_java_entity_dd_suffix = EJB
-# ij_java_entity_eb_suffix = Bean
-# ij_java_entity_hi_suffix = Home
-# ij_java_entity_lhi_prefix = Local
-# ij_java_entity_lhi_suffix = Home
-# ij_java_entity_li_prefix = Local
-# ij_java_entity_pk_class = java.lang.String
-# ij_java_entity_vo_suffix = VO
-# ij_java_enum_constants_wrap = off
-# ij_java_extends_keyword_wrap = off
-# ij_java_extends_list_wrap = off
-# ij_java_field_annotation_wrap = split_into_lines
-# ij_java_finally_on_new_line = false
-# ij_java_for_brace_force = never
-# ij_java_for_statement_new_line_after_left_paren = false
-# ij_java_for_statement_right_paren_on_new_line = false
-# ij_java_for_statement_wrap = off
-# ij_java_generate_final_locals = false
-# ij_java_generate_final_parameters = false
-# ij_java_if_brace_force = never
-ij_java_imports_layout = org.apache.flink.**, |, org.apache.flink.shaded.**, |, *, |, javax.**, |, java.**, |, scala.**, |, $*
```

Comment:
replace flink with numaflow and enforce this configuration? or default organize imports step is fine?

Reply:
default should be fine..